print/enscript/Portfile

```diff
@@ -28,6 +28,7 @@ depends_lib         port:gettext
 # http://www.freebsd.org/cgi/cvsweb.cgi/ports/print/enscript-letter/
 # and from http://support.rubyforge.org/svn/trunk/support/ruby.st
 patchfiles-append   patch-ruby_syntax.diff
+patchfiles          patch-string_h.diff
```

Comment:
using `patchfiles` here will overwrite (and thus ignore) the patch on the previous line; instead do
```
patchfiles-append   patch-ruby_syntax.diff \
                    patch-string_h.diff
```

Reply:
Done, also added the size to make lint happier.